Libraries/Components/Button.js

```diff
@@ -18,7 +18,7 @@ const Text = require('../Text/Text');
 const TouchableNativeFeedback = require('./Touchable/TouchableNativeFeedback');
 const TouchableOpacity = require('./Touchable/TouchableOpacity');
 const View = require('./View/View');
-
+import type {AccessibilityState} from './View/ViewAccessibility';
```

Comment:
Nit: should this also be grouped with line 24? I can also fix this on import

Reply:
Yes, this should be grouped with other typed imports 💯. Please fix this during import